case_studies/cherry_hill_high_school_east/2015-09-21_c9_and_ajar.io_part_1/recap.md

```diff
@@ -3,7 +3,7 @@
 ![CHE CS at work](http://i.imgur.com/249LPMm.jpg?1)
 
 > *Dear reader*, these recaps were initially requested by
-> [hackedu](https://hackedu.us/) for other club organizers to read. However if
+> [hackedu](https://hackclub.io/) for other club organizers to read. However if
```

Comment:
I decided to only change the domains in case studies to preserve history. Do you think that's a good idea?